python/ffsim/protocols/trace_protocol.py

```diff
@@ -45,3 +51,72 @@ def trace(obj: Any, norb: int, nelec: tuple[int, int]) -> float:
         "The object did not have a _trace_ method that returned the trace "
         "or a _diag_ method that returned its diagonal entries."
     )
+
+
+def _trace_term(
+    op: tuple[tuple[bool, bool, int], ...], norb: int, nelec: tuple[int, int]
+):
+    n_elec_alpha, n_elec_beta = nelec
+
+    combined_indices = set([(orb, spin) for _, spin, orb in op])
```

Comment:
The square brackets are unnecessary. Also sorry to be annoying but please keep the ordering of the tuple the same, i.e. `(spin, orb)` instead of `(orb, spin)`. Also, this variable name is pretty opaque, I suggest `spin_orbs` instead.